service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Party.java

```diff
@@ -21,4 +23,8 @@
     public final Address address;
     public final EmailAddress email;
     public final Telephone telephoneNumber;
+
+    public String buildFullName() {
```

Comment:
Not sure what your experience but word `builder` does not work well for me. There is no building process, just simple concatenation. A `getFullName` works much better for me. Please rename it.

Reply:
your right @geek-soft `getFullname` will be preferred but given we are using Lombok, using `get` prefix will try to deserialize a `fullName` field which does not exist and will result in a `Unrecognized field "fullName"`.

Reply:
Rather than an issue with lombok, it more sounds like you are describing an issue with jackson seeing as lombok doesn't deal with serialisation and deserialisation

Reply:
just thinking about it again, we can add `@JsonIgnore` to the `getFullName` method